kubernetes/samples/scripts/create-fmw-infrastructure-domain/wlst/create-domain-script.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+export DOMAIN_HOME=${DOMAIN_HOME_DIR}
+
+# Create the domain
+wlst.sh -skipWLSModuleScanning \
+        /u01/weblogic/createFMWDomain.py \
+        -oh /u01/oracle \
+        -jh /usr/java/latest \
+        -parent ${DOMAIN_HOME}/.. \
+        -name ${CUSTOM_DOMAIN_NAME} \
+        -user `cat /weblogic-operator/secrets/username` \
+        -password `cat /weblogic-operator/secrets/password` \
+        -rcuDb ${CUSTOM_CONNECTION_STRING} \
+        -rcuPrefix ${CUSTOM_RCUPREFIX} \
+        -rcuSchemaPwd `cat /weblogic-operator/rcu-secrets/password` \
```

Comment:
Per note in Slack, seems the username is note used as it's formed in the domain script... I presume the RCU secret creation script would be adjusted...

The other question is would you also want sys users/pass in the RCU utility is not run from k8s Job?

Reply:
its there for future use